ISSUE_TEMPLATE/next.md

```diff
@@ -62,13 +74,18 @@ fedora-coreos-stream-generator -releases=https://fcos-builds.s3.amazonaws.com/pr
 - [ ] Wait for the PR to be approved.
 - [ ] Once approved, merge it and verify that the [`sync-stream-metadata` job](https://jenkins-fedora-coreos.apps.ocp.ci.centos.org/job/sync-stream-metadata/) syncs the contents to S3
 - [ ] Verify the new version shows up on [the download page](https://getfedora.org/en/coreos/download?stream=next)
+    - [ ] x86_64
+    - [ ] aarch64
 - [ ] Verify the incoming edges are showing up in the [update graph](https://builds.coreos.fedoraproject.org/graph?stream=next)
+    - [ ] x86_64
+    - [ ] aarch64
```

Comment:
@lucab - does the update graph web interface dropdown for architecture work? I couldn't get it to work.

Reply:
The event listeners for the dropdowns are not wired in, but URL query parameters should work by appending `&basearch=aarch64`: https://builds.coreos.fedoraproject.org/graph?stream=next&basearch=aarch64

Reply:
At least https://builds.coreos.fedoraproject.org/graph?basearch=aarch64&stream=next works

Reply:
ok cool will update.. side note.. loving this update graph 🤣: 

![Screenshot 2021-09-15 at 06-26-20 Fedora CoreOS updates graph](https://user-images.githubusercontent.com/4530030/133417171-4027b843-4e7a-4fe6-bb0b-78bb9d0b9d86.png)